src/app/modules/core/pipes/uint-array-to-hex-string.pipe.ts

```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'uintArrayToHexString'
+})
+export class UintArrayToHexStringPipe implements PipeTransform {
```

Comment:
This should have comprehensive tests, but we can do it in a separate PR as this one has been open for a bit